src/pages/contest.astro

```diff
@@ -0,0 +1,80 @@
+---
+import BaseLayout from "../components/Common/BaseLayout.astro";
+---
+
+<BaseLayout pageTitle="Contest">
+    <div class="container my-5">
+      <div class="row">
+        <div class="col-12">
+            <div class="image-container text-center">
+                <img src="../../public/assets/images/contest/contest.png" alt="Large Event Image" class="img-fluid w-100"/>
+                <p class="image-footer lead mt-3">
+                    DevFest Bari will be not only full of contents but also full of emotions, and we want to collect them all to review for the next days and never forget them🚀
+                </p>
+            </div>
+        </div>
+      </div>
+  
+      <div class="row mt-4">
+        <div class="col-lg-6 d-flex flex-column justify-content-center">
+          <h3 class="font-medium text-lg my-2">How it Works?</h3>
+          <p>
+            Post your photos on socials using <strong>#DevFestBari23</strong> on December 2nd October tagging us!
+          </p>
+          <p>
+            <p>
+                You can choose to do this challenge on your preferred social, just remember to tag <strong>@gdgbari</strong> on
+                <a class="text-amber-500" href="https://twitter.com/gdgbari" target="_blank" rel="noopener noreferrer">Twitter</a>, 
```

Comment:
Al posto dell'URL di Twitter, sarebbe preferibile introdurre `TWITTER_PAGE` in `WebsiteConfig` e usarla qui.